### PR TITLE
Update windowsauth.md

### PR DESCRIPTION
**Pull request by Rick-Anderson**

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@ardalis  ready for review

File: aspnetcore/security/authentication/windowsauth.md [hunk 1]
```diff
@@ -20,33 +20,38 @@ Windows authentication can be configured for ASP.NET Core apps hosted with IIS o
...
+The Visual Studio Web Application template can be configured to support Windows authentication.
 
-### Installing the Windows authentication app template
+### Using the Windows authentication app template
 
-In Visual Studio, create a new ASP.NET Core Web Application. Next, select Web Application from the list of templates, and click the Change Authentication button. Choose Windows Authentication from the dialog that appears.
+In Visual Studio:
+* Create a new ASP.NET Core Web Application. 
+* Select Web Application from the list of templates.
+* Select the Change Authentication button and  select **Windows Authentication**.
```
Comment (ardalis) [inline aspnetcore/security/authentication/windowsauth.md]: Extra space "and  select"

File: aspnetcore/security/authentication/windowsauth.md [hunk 2]
```diff
@@ -87,11 +92,11 @@ Using Visual Studio or the dotnet CLI, *publish* your app to the destination fol
 
 Learn more about [publishing to IIS](https://docs.microsoft.com/en-us/aspnet/core/publishing/iis).
 
-After completing these steps, you should be able to launch your app from its IIS-bound path and Windows authentication should work correctly.
+Launch the app to verify Windows authentication is working.
 
 ## Enabling Windows authentication with WebListener
 
-Although Kestrel doesn't support Windows authentication, you can use [WebListener](/fundamentals/servers/weblistener) to support self-hosted scenarios on Windows. Use the following code snippet to configure your app's web host to use WebListener with Windows authentication:
+Although Kestrel doesn't support Windows authentication, you can use [WebListener](/fundamentals/servers/weblistener) <!-- BAD - xref only!!!! --> to support self-hosted scenarios on Windows. The following example configures the app's web host to use WebListener with Windows authentication:
```
Comment (ardalis) [inline aspnetcore/security/authentication/windowsauth.md]: What is xref? Searched CONTRIBUTING.md and didn't find any reference to it. What should this link be?

File: aspnetcore/security/authentication/windowsauth.md [hunk 3]
```diff
@@ -118,6 +123,13 @@ public class Program
...
+
+Don't you want something like:
+
+An app can allow anon access with Win auth using  [AllowAnonymous] and Authorize
+-->
+
 ### Impersonation
 
-ASP.NET Core does not implement impersonation. Your app will run with its application identity for all requests, using app pool or process identity. If you need to explicitly perform an action on behalf of a user, you should use ``WindowsIdentity.RunImpersonated``. Run a single action in this conext and then close the context. Note that ``RunImpersonated`` does not support async and should not be used for complex scenarios (for example, wrapping entire requests or middleware chains is not supported or recommended).
+ASP.NET Core does not implement impersonation. Apps run with the application identity for all requests, using app pool or process identity. If you need to explicitly perform an action on behalf of a user, use ``WindowsIdentity.RunImpersonated``. Run a single action in this conext and then close the context. Note that ``RunImpersonated`` does not support async and should not be used for complex scenarios. For example, wrapping entire requests or middleware chains is not supported or recommended.
```
Comment (ardalis) [inline aspnetcore/security/authentication/windowsauth.md]: "conext"

File: aspnetcore/security/authentication/windowsauth.md (hunk 2 above)
Comment (Rick-Anderson) [inline aspnetcore/security/authentication/windowsauth.md]: [link_text](xref:uid_of_the_topic#bookmark)
get the UID from the meta-data

**Pull request merged**